lxd/instance/drivers/driver_qemu_templates.go

```diff
@@ -23,7 +23,7 @@ type = "pseries"
 type = "s390-ccw-virtio"
 {{end -}}
 accel = "kvm"
-usb = "off"
+usb = "on"
```

Comment:
@monstermunchkin as this will affect all VMs, what does this setting enable? Thanks

Reply:
It enables USB for guests. Without it, USB pass through won't work as it won't be able to find a USB bus.